### DISCUSSION TITLE
Coverting JSON to mask with shape_type='mask'

### DISCUSSION THREAD

**Discussion by AmaiaGaste** [wkentaro/labelme#1434] [Q&A / Help]
I tried to use the labelme.utils.shape_to_mask function and I get the following error: AssertionError: Polygon must have points more than 2.

It seems there is an assert that checks if there is more than 2 points and my JSON only as 2 points and a mask. I want to convert the JSON into a PNG or JPG mask so that I can use the annotated image as ground truth. Thank you!

**Comment by ForLouis** [comment]
I'm having this problem too. How can I fix it?

**Reply by wkentaro** [reply]
https://github.com/wkentaro/labelme/pull/1560 should have resolved this issue. Can you try if it's resolved with labelme==5.8.1?

<img width="999" alt="image" src="https://github.com/user-attachments/assets/212ecdc2-fd4a-4f28-85b1-819db4bbda93" />
